test/test-options.js

```diff
@@ -95,21 +95,19 @@ $(function() {
     
     test("Tooltip options", function(assert) {
         var tooltip_async_done = assert.async(CST_NB_OF_FRANCE_DPTMT);
-        var container_class = "CONTAINER_CLASSNAME";
         var tooltip_class = "TOOLTIP_CLASSNAME";
         var additional_prop = {
-            "border-left": "5"
+            "border-left": "5px solid rgb(0, 255, 0)"
```

Comment:
Yes, I was having trouble with this test. I left it "as is" to be corrected later ;-)